### ISSUE TITLE
Webdriver test failures for collapsible menu view

### ISSUE THREAD

**Issue by nn052161** [cerner/terra-framework#867]
# Bug Report
Webdriver test failures for collapsible menu view tests in a few scenarios.

## Description
We pass 'CollapsibleMenuView.ItemGroup' which consists of 'CollapsibleMenuView.Item' as subMenuItems to 'CollapsibleMenuView.Item' [Project code](https://github.cerner.com/AcuteCaseManagementUI/case-management-notes-js/blob/feature/ACM-463/src/case-management-notes/components/CaseManagementNotesHeader.jsx#L118).
 [Wdio tests](https://github.cerner.com/AcuteCaseManagementUI/case-management-notes-js/blob/feature/ACM-463/tests/wdio/case-management-notes-spec.js#L137) for that 'CollapsibleMenuView.ItemGroup' that is failing.

## Steps to Reproduce

1. Clone [github url](https://github.cerner.com/AcuteCaseManagementUI/case-management-notes-js/tree/feature/ACM-463)
2. Run 'npm run test:wdio:local'

## Additional Context / Screenshots
![Wdio-Error](https://user-images.githubusercontent.com/54286765/64307094-a2644f80-cf5a-11e9-99d5-eff9b6e99e84.png)

## Expected Behavior
Test should pass without failures

## Environment

* Component Name and Version: case-management-notes-column
* Browser Name and Version: Chrome
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version: selenium V3.14 
* Operating System and version (desktop or mobile): Mac os

## @ Mentions
@dkasper-was-taken 
@jmsv6d

**Comment by bjankord** [comment]
@dkasper-was-taken has some info on how to get to this scenario. We'll work on getting it in a branch for further review.

**Comment by nramamurth** [comment]
@nn052161 Do you have an update on it?

**Comment by nn052161** [comment]
Yup, updating terra-collapsible-menu-view to the latest version fixed the issue. Thanks

**Comment by nramamurth** [comment]
Based on the above discussion, this issue is being closed. Please feel free to reopen if this occurs again.
